src/the-2021-scheduler.md

```diff
@@ -7,7 +7,7 @@ scheduler](./the-anki-2.1-scheduler.md) ("v2").
 
```

Comment:
This is a long-time peeve of mine, so as long as we're here ...

Since "v3" is the name that stuck, and it's the name in the left nav bar, can we change the page headline to "The Anki v3 scheduler"? That won't turn "the-2021-scheduler.html" into a deadlink, will it? Preserving links to this page is much more important!

Reply:
Below the header I'd still like "2021 scheduler" being mentioned. A little bit of Anki trivia won't hurt. This is something we would also want to do for "2.1 scheduler" (in this case, the manual will also need correcting).

Reply:
I'm totally on board with leaving the sentence ("The 2021 scheduler ("v3") is an update to ...") as-is. Just the title.